businessCentral/app/src/ScheduleTaskAssignment.Report.al

```diff
@@ -0,0 +1,149 @@
+report 82561 "ADLSE Schedule Task Assignment"
+{
+    ApplicationArea = Basic, Suite;
+    Caption = 'Schedule Export to Azure Data Lake';
```

Comment:
Maybe another caption. Because you can also export to Fabric.
Like: "Schedule Export"